mono/mini/driver.c

```diff
@@ -1988,7 +1988,8 @@ mono_main (int argc, char* argv[])
 		} else if (strcmp (argv [i], "--hybrid-aot") == 0) {
 			mono_jit_set_aot_mode (MONO_AOT_MODE_HYBRID);
 		} else if (strcmp (argv [i], "--full-aot-interp") == 0) {
```

Comment:
I would love to take this occasion to remove this command line parameter which seems like a hack to me while it's equivalent to `--full-aot --interpreter`, but that's not required for this PR.